Tests/TensorFlowTests/TensorAutoDiffTests.swift

```diff
@@ -153,6 +153,42 @@ final class TensorAutoDiffTests: XCTestCase {
         XCTAssertEqual(varianceGradAlongAxes(input), expected)
     }
 
+    func testMin() {
```

Comment:
Please add the corresponding Python TensorFlow (2.0) reproducer in a block comment for both min and max tests.

Reply:
Here's a start:
```python
import tensorflow as tf

tf.enable_eager_execution()

def test_op(x, y, op):
  with tf.GradientTape(persistent=True) as t:
    t.watch(x)
    t.watch(y)
    result = tf.reduce_sum(op(x, y))
  
  print(t.gradient(result, x))
  print(t.gradient(result, y))
  
  with tf.GradientTape(persistent=True) as t:
    t.watch(x)
    t.watch(y)
    result = tf.reduce_sum(op(y, x))
  
  print(t.gradient(result, x))
  print(t.gradient(result, y))

x = tf.constant([4, 5, 3], dtype=tf.float32)
y = tf.constant([4, 2, 6], dtype=tf.float32)
test_op(x, y, tf.math.maximum)
test_op(x, y, tf.math.minimum)

x = tf.constant([1, 5, 3], dtype=tf.float32)
y = tf.constant([4], dtype=tf.float32)
test_op(x, y, tf.math.maximum)
test_op(x, y, tf.math.minimum)

x = tf.constant([1, 5, 3], dtype=tf.float32)
y = tf.constant(2, dtype=tf.float32)
test_op(x, y, tf.math.maximum)
test_op(x, y, tf.math.minimum)
```

Reply:
I did the following in colab:
```
with tf.GradientTape() as t:
  t.watch([a, b])
  y = tf.math.reduce_sum(tf.minimum(a, b))
grads = t.gradient(y, [a, b])
```
which reproduced the discrepancies in the `swift-api` version which averages the ties.